apps/web/src/ee/clerk/ClerkRoutes.tsx

```diff
@@ -0,0 +1,35 @@
+import { SignedIn, SignedOut } from '@clerk/clerk-react';
+import { Route } from 'react-router-dom';
+import { PublicPageLayout } from '../../components/layout/components/PublicPageLayout';
+import { ROUTES } from '../../constants/routes';
+import OrganizationListPage from './pages/OrganizationListPage';
+import QuestionnairePage from './pages/QuestionnairePage';
+import SignInPage from './pages/SignInPage';
+import SignUpPage from './pages/SignUpPage';
+
+export const ClerkRoutes = () => {
+  return (
+    <>
+      <Route
+        element={
+          <SignedOut>
+            <PublicPageLayout />
+          </SignedOut>
+        }
+      >
+        <Route path={ROUTES.AUTH_SIGNUP + '/*'} element={<SignUpPage />} />
+        <Route path={ROUTES.AUTH_LOGIN + '/*'} element={<SignInPage />} />
+      </Route>
+      <Route
+        element={
+          <SignedIn>
+            <PublicPageLayout />
+          </SignedIn>
+        }
+      >
+        <Route path={ROUTES.AUTH_SIGNUP_ORGANIZATION_LIST} element={<OrganizationListPage />} />
```

Comment:
Why is this route necessary? The SignUp component should display this page since organizations are enabled for our instance.

Reply:
The `OrganizationListPage` is necessary otherwise it would open Clerk site with the component, the route is probably not not necessary but when I do `ROUTES.AUTH_SIGNUP + '/*'` (or `'auth/signup/*'` it doesn't work for some unknown reason, the only format it works is `'/auth/singup/:something`, otherwise it wont display the org select list at all